jobs/rocoto/gempak.sh

```diff
@@ -1,15 +1,24 @@
 #! /usr/bin/env bash
 
 source "${HOMEgfs}/ush/preamble.sh"
-
-###############################################################
-. "${HOMEgfs}/ush/load_fv3gfs_modules.sh"
+source "${HOMEgfs}/ush/load_fv3gfs_modules.sh"
 status=$?
 if (( status != 0 )); then exit "${status}"; fi
 
 export job="gempak"
 export jobid="${job}.$$"
 
+source "${HOMEgfs}/ush/jjob_header.sh" -e "gempak" -c "base gempak"
+
+# shellcheck disable=SC2153
+fhr3="${FHR3}"
+fhr=$(( 10#${fhr3} ))
+if (( fhr > FHMAX_GFS )); then
+  echo "Nothing to process for FHR = ${fhr3}, cycle"
+fi
+
+export fcsthrs="${fhr3}"
```

Comment:
Can remove all this and just update scripts to use `FHR3` instead of `fcsthrs`. Calling the j-job header will no longer be necessary because we won't need the value of `FHMAX_GFS`.
```suggestion
```

Reply:
Lines 13-20 are deleted.